packages/feedback/src/widget/Icon.ts

```diff
@@ -0,0 +1,53 @@
+import { setAttributesNS } from '../util/setAttributesNS';
+
+const SIZE = 20;
+const XMLNS = 'http://www.w3.org/2000/svg';
+
+interface IconReturn {
+  $el: SVGElement;
+}
+
+/**
+ * Feedback Icon
+ */
+export function Icon(): IconReturn {
+  const cENS = document.createElementNS.bind(document, XMLNS);
```

Comment:
is `cENS` --> `createElementNS`? what about calling this e.g. `root` or something like this?

Reply:
ah wait, nvm, we are binding this only 🤔 then I'd just call this `createElementNS`, which is a bit clearer!